test/Bart/Jenkins/Job_Test.php

```diff
@@ -1,113 +1,68 @@
 <?php
 namespace Bart\Jenkins;
 
-use Bart\Diesel;
-use Bart\Curl;
+use Bart\BaseTestCase;
 
-class Job_Test extends \Bart\BaseTestCase
+class Job_Test extends BaseTestCase
 {
-	public static $domain = 'www.norris.com';
-	public static $job_name = 'chuck norris';
-
-	/**
-	 * Mock the metadata returned by curl
-	 */
-	public function configure_for_health_tests($last_success, $last_completed)
+	public static $projectPath = 'job/chuck norris';
```

Comment:
You should test this with a job in a sub-project, to ensure that your `explode` and `urlencode` code works.